code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -478,7 +478,7 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 /obj/item/restraints/handcuffs/energy
 	name = "hard-light energy field"
 	desc = "A hard-light field restraining the hands."
-	icon_state = "cuff_white" // Needs sprite
+	icon_state = "cuff" // Needs sprite
```

Comment:
Needs sprite?

Reply:
Its always needed a sprite - thats entirely separate to the thing at hand.